GitVersionCore.Tests/Mocks/MockCommitLog.cs

```diff
@@ -8,7 +9,10 @@ public class MockCommitLog : ICommitLog, ICollection<Commit>
 
     public IEnumerator<Commit> GetEnumerator()
     {
-        return Commits.GetEnumerator();
+        if (SortedBy == CommitSortStrategies.Reverse)
```

Comment:
I'm not sure to understand why you'd like it reversed. Could you please me a little bit more context?

FWIW, `CommitSortStrategies.Reverse` will take more time as the whole log has to be walked before yielding the first result. The default mimics the git.git `git log` behavior that makes sense most of the time and allow to stream the results as the log is walked.